File: app/src/main/java/workshop/akbolatss/tagsnews/screen/board/BoardView.java
package workshop.akbolatss.tagsnews.screen.board;

import java.util.List;

import workshop.akbolatss.tagsnews.base.BaseView;
import workshop.akbolatss.tagsnews.model.dao.RssSource;


public interface BoardView extends BaseView {

    void onInitSources(List<RssSource> rssSources);

    void onUpdateSources(List<RssSource> rssSources);
}

File: app/src/main/java/workshop/akbolatss/tagsnews/screen/board/BoardView.kt
package workshop.akbolatss.tagsnews.screen.board

import workshop.akbolatss.tagsnews.base.BaseView
import workshop.akbolatss.tagsnews.model.dao.RssSource

/**
 * MVP View for #BoardActivity
 * @see BoardActivity
 */
interface BoardView : BaseView {

    fun onInitSources(rssSources: List<RssSource>)

    fun onUpdateSources(rssSources: List<RssSource>)
}
